### ISSUE TITLE
Transitions - explore how to add ticks for seamless transitions

### ISSUE THREAD

**Issue by elsa-brown** [space-chat/space-chat#59]

**Comment by s-i-s** [comment]
This is a research-based issue, looking into how to use React/Redux (+etc) to implement tick-based animations. Here's a list of what I'll be looking at to start --
Reading: 
- https://facebook.github.io/react/docs/animation.html
- https://github.com/chenglou/react-motion
- https://facebook.github.io/react-native/docs/animations.html#content
- https://tech.instacart.com/animating-programmatically-with-redux-and-react-e304965ec828
- https://jsfiddle.net/dcocchia/4kuxhrhL/45/
- https://www.reddit.com/r/reactjs/comments/3vw7xq/reduxflux_animation_confusion/
- https://swizec.com/blog/animating-with-react-redux-and-d3/swizec/6775
- http://swizec.com/reactd3js/ (not sure yet how relevant)
- ticks in gaming development - https://www.reddit.com/r/explainlikeimfive/comments/4mn531/eli5_what_is_atick_in_gaming_development/

Examples:
- Keziyah's bubble code in bubbles-#62 branch
- Julie's Staged Fright app -- https://github.com/JulieBusch/staged-fright
- Transcend VR codebase
